axelor-account/src/main/resources/views/Charts.xml

```diff
@@ -532,17 +534,22 @@
       self.debit
```

Comment:
please, add a filter on the year field to take into account on the typeSelect = 1